cli/cmd/report_definitions_revert.go

```diff
@@ -0,0 +1,67 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2023, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package cmd
+
+import (
+	"strconv"
+
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+// revert command is used to rollback lacework report definition to a previous version
+var reportDefinitionsRevertCommand = &cobra.Command{
+	Use:     "revert <report_definition_id> <version>",
+	Aliases: []string{"restore"},
```

Comment:
I thought @marcosgm said to use `restore`? 🤔 - With alias to `revert`

The reason it is important to choose one is because the entire output of this command will say:
```
Restoring report definition from version 'X' ...
```
and
```
The report definition 'ABC' was restored to version 'X'
````